doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc

```diff
@@ -0,0 +1,60 @@
+[id='operator-deploy-pim-proc']
+= Setting Process Instance Migration configuration for the environment
+
+You can use the operator to deploy the Process Instance Migration (PIM) service. You can use the PIM service to define the migration between two different process definitions, known as a migration plan. You can then apply the migration plan to the running process instance in a specific KIE Server. 
+
+The PIM service uses a database server for its operation.
+
+.Prerequisites
+
+* You completed basic configuration of a {PRODUCT} environment using {operator_name} in the installer wizard according to the instructions in <<operator-deploy-basic-proc>>.
+
+.Procedure
+. If the *Installation*, *Security*, *Console*, *KIE Servers*, or *Smart Router* tab is open, click *Next* until you view the *Process Instance Migration* tab.
+. In the *Database type* field, select the database that the PIM service must use. The following values are available:
+** `mysql`: A MySQL server, created in a separate pod.
+** `postgresql`: A PostgreSQL server, created in a separate pod. Use this setting unless you have a specific reason to use any other setting.
+** `h2`: A built-in `h2` database engine that does not require a separate pod.
+** `external`: An external database server. 
+. If you selected any database except `external`, a Persistent Volume Claim will be created to store the database. Optionally, set configuration parameters of the persistent volume to create:
+** In the *Size* field, enter the size of the persistence volume
+** In the *StorageClass name* field, enter the storage class name for the persistent volume
+. If you selected the `external` database and you want to use any database server except PostgreSQL or MySQL, you must provide a {KIE_SERVER} extension image with the database server driver according to instructions in <<externaldb-build-proc>>. The PIM service can use the {KIE_SERVER} extension image. To configure the PIM service to use this extension image, make the following changes:
+.. Select the *Enable extension image stream* box.
```

Comment:
That is not part of the PIM deployment

Reply:
But it is in the UI for PIM. I will remove it from the doc but as mentioned above I'll make an RHPAM task about this.

Reply:
My mistake, sorry. It is not in the UI.

Reply:
The review request will be coming after I sort out dialects, as these need to come in here too.